components/ShoutBox/shoutbox.tsx

```diff
@@ -155,7 +145,7 @@ const Chat = ({ limit, isOpen }: ShoutBoxProps) => {
     <div className="mx-auto flex h-96 w-full max-w-6xl px-[25px] py-6 md:h-[38rem]">
       <div className="mx-auto my-0 h-auto w-full flex-wrap overflow-auto overflow-x-hidden shadow-md">
         <div
-          className="h-[81%] overflow-auto overflow-x-hidden px-0 py-2 text-white md:h-[85%]"
+          className="h-[81%] flex-col-reverse overflow-auto overflow-x-hidden px-0 py-2 text-white md:h-[85%]"
```

Comment:
Oho... Onks se oikeesti näin simppeli korjaus? :O

Reply:
Testasin develop-sivua firefoxilla duuniläbärsil eikä tää näyttäny mun mielestä toimivan...

Reply:
Pitäs käydä invalidoimassa kakku, mut joo on näin simppeli.